sdk/src/client/secret/mod.rs

```diff
@@ -602,3 +604,29 @@ where
 
     Ok(tx_payload)
 }
+
+#[async_trait]
+pub trait SignBlockExt {
```

Comment:
Can we just call it SignBlock? I think Ext would fit if it was a super-trait of SecretManage but it's not